### PR TITLE
ci: add diff link for previously deployed commit

### PR DESCRIPTION
**Pull request by jakubgs**
Devs are a fussy bunch.

### CODE REVIEW

**Review by felicio — APPROVED** [review]

File: ci/Jenkinsfile.release [hunk 1]
```diff
@@ -71,13 +71,20 @@ pipeline {
...
           title: "${env.JOB_NAME}#${env.BUILD_NUMBER}",
           description: """
             Nim-Waku deployment successful!
-            Image: [`${IMAGE_NAME}:${IMAGE_TAG}`](https://hub.docker.com/r/${IMAGE_NAME}/tags?page=1&name=${IMAGE_TAG})
-            Branch: [`${GIT_BRANCH.minus("origin/")}`](https://github.com/status-im/nwaku/commits/${GIT_BRANCH.minus("origin/")})
-            Commit: [`${GIT_COMMIT.take(8)}`](https://github.com/status-im/nwaku/commit/${GIT_COMMIT.take(8)})
+            Image: [`${IMAGE_NAME}:${IMAGE_TAG}`](https://hub.docker.com/r/${IMAGE_NAME}/tags?name=${IMAGE_TAG})
+            Branch: [`${repo.branch}`](${repo.url}/commits/${repo.branch})
+            Commit: [`${repo.commit}`](${repoUrl}/commit/${repo.commit})
+            Previous: [`${repo.prev}`](${repo.url}/compare/${repo.prev}...${repo.commit})
```
Comment (felicio) [inline ci/Jenkinsfile.release:87]: Non-blocking: `Previous` > `Comparison`

Comment (felicio) [reply ci/Jenkinsfile.release:87]: `[`${repo.prev}`]` > [`${repo.prev}...${repo.commit}`]

Comment (jakubgs) [reply ci/Jenkinsfile.release:87]: I don't think `Comparison` is a good word, since it doesn't actually indicate what we're comparing with.

Comment (felicio) [reply ci/Jenkinsfile.release:87]: If written as `Comparison: 75f4c67...6df2317` I'd say so, but no biggie 🙂. Feel free this close this thread 👍 .

Comment (felicio) [reply ci/Jenkinsfile.release:87]: Other keyword: Diff, Changes

**Review by LNSD — APPROVED** [review]
LGTM ✅

**Pull request merged**